src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -113,30 +112,26 @@ private Future<Void> processInventory(CompositePoLine compPOL, RequestContext re
           logger.debug(String.format("Pieces were removed : %s", deletedIds));
         }
       })
-        .mapEmpty();
+      .mapEmpty();
     } else if (PoLineCommonUtil.isItemsUpdateRequired(compPOL)) {
```

Comment:
We should not delete check PoLineCommonUtil.isHoldingsUpdateRequired(compPOL)
Lets imageine situation when user in Create Inventory field choses INSTANCE_HOLDING_ITEM, so we need in some way to process holdings, and now we just skip this method and return succeeded future withot any modifactions